scripts/lab/cmaes.py

```
@@ -0,0 +1,503 @@
+                    rank = 1 + int(round((1 - qq) * 39))
+                    share = min(1.0, max(0.0, qq))
+                    fired = "sim:3,trust:1" if qq > 0.5 else "-"
+                    if alive:
+                        fh.write(f"== {sp} | spawn 0,0 (dry run) | Medium == FINAL rank={rank} share={share:.2f} alive=true tiles={tiles} troops=0k players=40 fired={fired}\n")
+                    else:
+                        fh.write(f"== {sp} | spawn 0,0 (dry run) | Medium == DEAD at 100s FINAL alive=false tiles=0 troops=0k players=40 fired={fired}\n")
+
+
+def score(results_dir, names, old=False, expect=30):
+    cmd = [sys.executable, SUMMARIZE] + (["--old-fitness"] if old else []) + ["--fitness", results_dir] + names
+    out = subprocess.check_output(cmd, text=True)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)